### PR TITLE
fix(cci): relocate the cci_pvc resource code to the services directory

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:
relocate the cci_pvc resource code to the services directory
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
relocate the cci_pvc resource code to the services directory
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cci" TESTARGS="-run TestAccCCIPersistentVolumeClaims_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cci -v -run TestAccCCIPersistentVolumeClaims_basic -timeout 360m -parallel 4
=== RUN   TestAccCCIPersistentVolumeClaims_basic
=== PAUSE TestAccCCIPersistentVolumeClaims_basic
=== CONT  TestAccCCIPersistentVolumeClaims_basic
--- PASS: TestAccCCIPersistentVolumeClaims_basic (55.76s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cci       55.803s

```

### CODE REVIEW

File: huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 
 	"github.com/chnsz/golangsdk/openstack/cci/v1/persistentvolumeclaims"
+
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/cci"
 )
 
 func TestAccCCIPersistentVolumeClaims_basic(t *testing.T) {
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go]:
```suggestion
func TestAccPersistentVolumeClaims_basic(t *testing.T) {
```
All package prefix need be removed.

File: huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go [hunk 2]
```diff
@@ -175,40 +178,40 @@ func testAccCheckCCIPersistentVolumeClaimsExists(n, volumeType string,
...
+			return "", fmt.Errorf("auto Scaling lifecycle hook not found: %s", pvc)
 		}
 		if pvc.Primary.Attributes["volume_type"] == "" || pvc.Primary.ID == "" {
-			return "", fmtp.Errorf("Unable to find the resource by import infos: %s/%s/%s",
-				HW_CCI_NAMESPACE, pvc.Primary.Attributes["volume_type"], pvc.Primary.ID)
+			return "", fmt.Errorf("unable to find the resource by import infos: %s/%s/%s",
+				acceptance.HW_CCI_NAMESPACE, pvc.Primary.Attributes["volume_type"], pvc.Primary.ID)
 		}
-		return fmt.Sprintf("%s/%s/%s", HW_CCI_NAMESPACE, pvc.Primary.Attributes["volume_type"], pvc.Primary.ID), nil
+		return fmt.Sprintf("%s/%s/%s", acceptance.HW_CCI_NAMESPACE, pvc.Primary.Attributes["volume_type"], pvc.Primary.ID), nil
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go]: variables suggestion:
```
namespace := acceptance.HW_CCI_NAMESPACE
volumeType := pvc.Primary.Attributes["volume_type"]
pvcId := pvc.Primary.ID
```

File: huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go (hunk 1 above)
Comment (jinyangyang222) [reply huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go]: fixed.

File: huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go (hunk 2 above)
Comment (jinyangyang222) [reply huaweicloud/services/acceptance/cci/resource_huaweicloud_cci_pvc_test.go]: fixed.

**Pull request merged**